tests/features/common/kie-kieserver-common.feature

```diff
@@ -21,14 +21,6 @@ Feature: Kie Server common features
       | password        | kieserver1!           |
       | expected_phrase | SUCCESS               |
 
-  Scenario: Configure kie server to be immutable, disable management and set startup strategy
```

Comment:
Just out of interest, is there any replacement for this test?

Reply:
yes, actually there were duplicated tests which I removed.
The removed tests sits on kie-kieserver-common.feature.

Reply:
ok, thanks